### PR TITLE
Adding documentation on how to remove a resource safely

### PR DESCRIPTION
**Pull request by djw8605**

### CODE REVIEW

File: docs/common/registration.md [hunk 1]
```diff
@@ -48,6 +48,24 @@ methods:
 
 For definitions for the various fields, consult the corresponding template file for the type of data you are updating.
 
+To remove or disable a resource already registered, follow the steps:
+
+1. If the resource is a Compute Element or OSG submitter, set `Active: false` within resource group's yaml file.  For example, to remove
+   the resource GLOW, edit the file `topology/University of Wisconsin/GLOW/GLOW.yaml`, setting `Active: false`:
+    
+        :::toml
```
Comment (brianhlin) [inline docs/common/registration.md]: Does this look any different as YAML?

File: docs/common/registration.md [hunk 2]
```diff
@@ -48,6 +48,24 @@ methods:
...
+        ...
+        Production: true
+        Resources:
+          GLOW:
+            Active: %RED%false%ENDCOLOR%
+            ...
+    
+    You may have to add the `Active` attribute it if does not already exist within the resource definition.
+
+2. If the resource is not an Compute Element or OSG submitter, you may remove the resource completely.  If there are no more resources
```
Comment (brianhlin) [inline docs/common/registration.md]: > not a Compute Element

File: docs/common/registration.md [hunk 3]
```diff
@@ -48,6 +48,24 @@ methods:
 
 For definitions for the various fields, consult the corresponding template file for the type of data you are updating.
 
+To remove or disable a resource already registered, follow the steps:
```
Comment (brianhlin) [inline docs/common/registration.md]: Just drop `, follow the steps`

File: docs/common/registration.md (hunk 1 above)
Comment (djw8605) [reply docs/common/registration.md]: Shoot, was working on another project that used toml.  Yaml kinda works, but the %RED% stuff doesn't work when I use `:::yaml`.  So I just removed the definition line entirely.

**Pull request merged**